net.solarnetwork.node.weather.ibm.wc/src/net/solarnetwork/node/weather/ibm/wc/HourlyDatumPeriod.java

```diff
@@ -0,0 +1,38 @@
+
+package net.solarnetwork.node.weather.ibm.wc;
+
+public enum HourlyDatumPeriod implements DatumPeriod {
+
+	SIXHOUR("6hour"),
+	TWELVEHOUR("12hour"),
+	ONEDAY("1day"),
+	TWODAY("2day"),
+	THREEDAY("3day"),
+	TENDAY("10day"),
+	FIFTEENDAY("15day");
+
+	private String period;
+
+	HourlyDatumPeriod(String period) {
+		this.period = period;
+	}
+
+	public String getPeriod() {
+		return this.period;
+	}
+
+	@Override
+	public String toString() {
+		return this.getPeriod();
+	}
+
+	static HourlyDatumPeriod getValue(String period) {
```

Comment:
I think this should be `public` and we have a convention in SN of naming static enum factory methods like `forX(Y)`, e.g.

```java
public static HourlyDatumPeriod forPeriod(String period) {
```